### PR TITLE
Task 2 commit

### PR DESCRIPTION
**Pull request by AleksejC**
Completed task 2 for my dibil-teacher.

### CODE REVIEW

File: src/ToDoList.css [hunk 1]
```diff
@@ -38,4 +40,72 @@ body {
     margin-left: 1620px;
     margin-top: 870px;
 }
-
+.newTaskBox {
```
Comment (dimitryc-ytree) [inline src/ToDoList.css]:
```suggestion
.new-task-box {
```
Change it everywhere

File: src/ToDoList.tsx [hunk 2]
```diff
@@ -1,7 +1,21 @@
 import React from 'react';
+import {useState} from "react";
+import Modal from '@mui/material/Modal';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import TextField from '@mui/material/TextField';
 import './ToDoList.css';
 
 export const ToDoList = () => {
+    const [openWindow, setOpenWindow] = useState(false);
```
Comment (dimitryc-ytree) [inline src/ToDoList.tsx]: bool variables' names should start with `is`, `are`, `has`, `have` prefixes
`openWindow ` seems like function name

File: src/ToDoList.tsx [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+
+    const addNewTaskButton = () => {
+        if (openWindow) {
+            setOpenWindow(false);
+        } else {
+            setOpenWindow(true);
+        }
+    }
```
Comment (dimitryc-ytree) [inline src/ToDoList.tsx]:
```suggestion
    const addNewTaskButton = () => {
        setOpenWindow(!openWindow);
    }
```

**Comment by AleksejC** [comment]
Снимок экрана (11)
Снимок экрана (12)

File: src/ToDoList.tsx (hunk 3 above)
Comment (dimitryc-ytree) [reply src/ToDoList.tsx]: Also `addNewTaskButton ` seems like a bit weird name
Possible alternatives:

- just `onClick`
- `onAddNewTaskClick`
- `toggleAddNewTaskModal`

Comment (dimitryc-ytree) [reply src/ToDoList.tsx]: Also you should use `useCallback` here

File: src/ToDoList.tsx [hunk 4]
```diff
@@ -26,7 +40,46 @@ export const ToDoList = () => {
...
+                aria-labelledby="modal-modal-title"
+                aria-describedby="modal-modal-description"
+            >
+                <Box className='newTaskBox'>
+                    <Typography id='modal-modal-title' className='title' variant='h4'>
+                        Adding new task
+                    </Typography>
+                    <Typography id='modal-modal-description' className='content'>
+                        <div className='contentDiv'>
+                            <p>New task name</p>
```
Comment (dimitryc-ytree) [inline src/ToDoList.tsx]: Tag `p` is not good choise here from semantic point of view
Use `label` instead

File: src/ToDoList.tsx [hunk 5]
```diff
@@ -26,7 +40,46 @@ export const ToDoList = () => {
...
+                            <TextField
+                                className='newTaskNameInput'
+                                id="standard-helperText"
+                                defaultValue="Task 1"
+                                helperText="0/100 characters"
+                                variant="standard"
+                                size='medium'
+                            />
+                            <p>Description</p>
```
Comment (dimitryc-ytree) [inline src/ToDoList.tsx]: Same

File: src/ToDoList.tsx [hunk 6]
```diff
@@ -26,7 +40,46 @@ export const ToDoList = () => {
...
+                                className='descriptionInput'
+                                id="outlined-multiline-static"
+                                multiline
+                                rows={9}
+                                helperText='0/1000 characters'
+                            />
+                        </div>
+                            <hr/>
+                            <div className='modalButtonsContainer'>
+                                <button className='modalButtons'>Cancel</button>
```
Comment (dimitryc-ytree) [inline src/ToDoList.tsx]: I guess you should implement it in this PR as well

**Review by dimitryc-ytree — CHANGES_REQUESTED** [review]
And delete your comment - without it I won't approve

File: src/ToDoList.css [hunk 7]
```diff
@@ -38,4 +40,127 @@ body {
...
+    border-radius: 0 0 25px 25px;
+}
+.content-div {
+    margin-left: 45px;
+}
+.new-task-name-input {
+    width: 500px;
+}
+label {
+    display: flex;
```
Comment (dimitryc-ytree) [inline src/ToDoList.css]: there is no display: flex required for children elements

**Review by dimitryc-ytree — APPROVED** [review]

**Review by dimitryc-ytree — CHANGES_REQUESTED** [review]
Fix display: flex

**Review by dimitryc-ytree — APPROVED** [review]

**Pull request merged**